### PR TITLE
Improve Exact Change readability

### LINKED ISSUE #10491: Exact Change
#### Challenge Name

https://www.freecodecamp.com/challenges/exact-change
#### Issue Description

It's impossible (so far as I know) to actually get the results requested in the instructions. For example: 

`checkCashRegister(3.26, 100.00, [["PENNY", 1.01], ["NICKEL", 2.05], ["DIME", 3.10], ["QUARTER", 4.25], ["ONE", 90.00], ["FIVE", 55.00], ["TEN", 20.00], ["TWENTY", 60.00], ["ONE HUNDRED", 100.00]]) should return [["TWENTY", 60.00], ["TEN", 20.00], ["FIVE", 15.00], ["ONE", 1.00], ["QUARTER", 0.50], ["DIME", 0.20], ["PENNY", 0.04]]`

The output it's looking for is actually:

`[ [ 'TWENTY', 60 ],  [ 'TEN', 20 ],  [ 'FIVE', 15 ],  [ 'ONE', 1 ],  [ 'QUARTER', 0.5 ],  [ 'DIME', 0.2 ],  [ 'PENNY', .04 ] ]`

If you attempt to format it with toFixed() the strings(which look more like the requested answers) will be rejected.
`
#### Browser Information

- Browser Name, Version: 
- Operating System: 
- Mobile, Desktop, or Tablet: 
#### Your Code

``` js



```

### PR DESCRIPTION
**Pull request by damon-myers**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #10491 

#### Description

Simply changes the bonfire "Exact Change" to use integers instead of decimal numbers in the test case description and expected results.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 1]
```diff
@@ -166,26 +166,26 @@
...
-        "// [\"TWENTY\", 60.00],",
-        "// [\"ONE HUNDRED\", 100.00]]",
+        "// [\"ONE\", 90],",
+        "// [\"FIVE\", 55],",
+        "// [\"TEN\", 20],",
+        "// [\"TWENTY\", 60],",
+        "// [\"ONE HUNDRED\", 100]]",
         "",
-        "checkCashRegister(19.50, 20.00, [[\"PENNY\", 1.01], [\"NICKEL\", 2.05], [\"DIME\", 3.10], [\"QUARTER\", 4.25], [\"ONE\", 90.00], [\"FIVE\", 55.00], [\"TEN\", 20.00], [\"TWENTY\", 60.00], [\"ONE HUNDRED\", 100.00]]);"
+        "checkCashRegister(19.50, 20, [[\"PENNY\", 1.01], [\"NICKEL\", 2.05], [\"DIME\", 3.10], [\"QUARTER\", 4.25], [\"ONE\", 90], [\"FIVE\", 55], [\"TEN\", 20], [\"TWENTY\", 60], [\"ONE HUNDRED\", 100]]);"
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Remove all trailing zeros that comes after decimal

- `checkCashRegister(19.50,`
- `[\"DIME\", 3.10]`

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 2]
```diff
@@ -166,26 +166,26 @@
...
...NDRED', val: 100},\n\t{ name: 'TWENTY', val: 20},\n\t{ name: 'TEN', val: 10},\n\t{ name: 'FIVE', val: 5},\n\t{ name: 'ONE', val: 1},\n\t{ name: 'QUARTER', val: 0.25},\n\t{ name: 'DIME', val: 0.10},\n\t{ name: 'NICKEL', val: 0.05},\n\t{ name: 'PENNY', val: 0.01}\n];\n\nfunction checkCashRegister(price, cash, cid) {\n    var change = cash - price;\n    var register = cid.reduce(function(acc, curr) {\n        acc.total += curr[1];\n        acc[curr[0]] = curr[1];\n        return acc;\n    }, {total: 0});\n    if(register.total === change) {\n        return 'Closed';\n    }\n    if(register.total < change) {\n        return 'Insufficient Funds';\n    }\n    var change_arr = denom.reduce(function(acc, curr) {\n        var value = 0;\n        while(register[curr.name] > 0 && change >= curr.val) {\n           change -= curr.val;\n           register[curr.name] -= curr.val;\n           value += curr.val;\n           change = Math.round(change * 100) / 100;\n        }\n        if(value > 0) {\n            acc.push([ curr.name, value ]);\n        }\n        return acc;\n    }, []);\n  if(change_arr.length < 1 || change > 0) {\n    return \"Insufficient Funds\";\n  }\n  return change_arr;\n}"
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Remove all trailing zeros that comes after decimal

- `{ name: 'DIME', val: 0.10}`

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 3]
```diff
@@ -166,26 +166,26 @@
...
-        "assert.deepEqual(checkCashRegister(19.50, 20.00, [[\"PENNY\", 0.50], [\"NICKEL\", 0], [\"DIME\", 0], [\"QUARTER\", 0], [\"ONE\", 0], [\"FIVE\", 0], [\"TEN\", 0], [\"TWENTY\", 0], [\"ONE HUNDRED\", 0]]), \"Closed\", 'message: <code>checkCashRegister(19.50, 20.00, [[\"PENNY\", 0.50], [\"NICKEL\", 0], [\"DIME\", 0], [\"QUARTER\", 0], [\"ONE\", 0], [\"FIVE\", 0], [\"TEN\", 0], [\"TWENTY\", 0], [\"ONE HUNDRED\", 0]])</code> should return \"Closed\".');"
+        "assert.isArray(checkCashRegister(19.50, 20, [[\"PENNY\", 1.01], [\"NICKEL\", 2.05], [\"DIME\", 3.10], [\"QUARTER\", 4.25], [\"ONE\", 90], [\"FIVE\", 55], [\"TEN\", 20], [\"TWENTY\", 60], [\"ONE HUNDRED\", 100]]), 'message: <code>checkCashRegister(19.50, 20, [[\"PENNY\", 1.01], [\"NICKEL\", 2.05], [\"DIME\", 3.10], [\"QUARTER\", 4.25], [\"ONE\", 90], [\"FIVE\", 55], [\"TEN\", 20], [\"TWENTY\", 60], [\"ONE HUNDRED\", 100]])</code> should return an array.');",
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Remove all trailing zeros that comes after decimal

- `[\"DIME\", 3.10]` <--- assertion
- `[\"DIME\", 3.10]` <-- message

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 4]
```diff
@@ -166,26 +166,26 @@
...
+        "assert.isString(checkCashRegister(19.50, 20, [[\"PENNY\", 0.01], [\"NICKEL\", 0], [\"DIME\", 0], [\"QUARTER\", 0], [\"ONE\", 0], [\"FIVE\", 0], [\"TEN\", 0], [\"TWENTY\", 0], [\"ONE HUNDRED\", 0]]), 'message: <code>checkCashRegister(19.50, 20, [[\"PENNY\", 0.01], [\"NICKEL\", 0], [\"DIME\", 0], [\"QUARTER\", 0], [\"ONE\", 0], [\"FIVE\", 0], [\"TEN\", 0], [\"TWENTY\", 0], [\"ONE HUNDRED\", 0]])</code> should return a string.');",
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Remove all trailing zeros that comes after decimal

- `checkCashRegister(19.50` <-- assertion
- `<code>checkCashRegister(19.50` <-- message

**Review by abhisekp — CHANGES_REQUESTED** [review]
@damon-myers Please remove all trailing zeros that comes after decimal

_There some more place that what i've marked._

Please fix the remaining issues and this is ready to merge :loop:

**Comment by damon-myers** [comment]
@abhisekp I was initially thinking we only wanted the change to apply to whole dollar amounts, but this makes more sense. On it now.

**Comment by raisedadead** [comment]
@damon-myers this has a a few unwanted commits. Mind doing a rebase against staging?

`git checkout <pr branch>`
`git pull --rebase upstream staging`
`git push origin --force`

Thanks.

**Comment by damon-myers** [comment]
@raisedadead, done and squashed to one commit.

**Review by abhisekp — APPROVED** [review]
<h1 align="center"> :sparkles:  :smile_cat:  :+1: LGTM :smiley: :+1:   :sparkles:  </h1>

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @abhisekp for the review and @damon-myers for the changes!

**Comment by damon-myers** [comment]
@raisedadead, @abhisekp Thanks for all of your help!

**Pull request merged**